### ISSUE TITLE
404: Not Found

### ISSUE THREAD

**Issue by bdeman00** [pssc/ha-addon-lms#69]
Settings menu;
sellected the clasic theme.

getting error if a sellected setings menu '404: Not Found' 

Current version: 8.3.1.60
home assist 11.5

**Comment by pssc** [comment]
@bdeman00 trying to recreate this and not having any luck, try restarting you web browser,  which browser are u using? any more details you can provide?

**Comment by pssc** [comment]
@bdeman00 ah are you using a there in LMS other than the default or material?

**Comment by bdeman00** [comment]
[image0.png][image1.jpeg][image2.jpeg][image4.png]

This are the steps I take.
LMS in my rapsberry pi 4
home assist;

  *
Core2024.2.1
  *   Supervisor2024.01.1
  *   Operating System11.5
  *   Frontend20240207.1


Op 11 feb 2024 om 18:44 heeft Phill (pssc) ***@***.***> het volgende geschreven:

﻿

@bdeman00<https://github.com/bdeman00> ah are you using a there in LMS other than the default or material?

**Comment by bdeman00** [comment]
Browser;
Firefox, web interface of home assist (app) and safari
Greetings Bart

Op 11 feb 2024 om 18:40 heeft Phill (pssc) ***@***.***> het volgende geschreven:

﻿

@bdeman00<https://github.com/bdeman00> trying to recreate this and not having any luck, try restarting you web browser, which browser are u using? any more details you can provide?

**Comment by rensdebee** [comment]
Hi, maybe this is completely unrelated but when using the material theme the icons don't load when loading from the HA sidebar (connecting to HA via direct IP also doesn't work). When loading up the page via IP:9000 icons do load.
![image](https://github.com/pssc/ha-addon-lms/assets/34692372/dbbd88df-4d17-4bbd-89cd-e04c3ec0a876)
![image](https://github.com/pssc/ha-addon-lms/assets/34692372/8615a99f-2d85-4262-947a-97569c6f4e91)

**Comment by pssc** [comment]
yes that will never work those urls don't look like they have been rewritten, but they can be an are on my test system for these icons  clear caches andrestar web browser and make sure skin material is selected in the addon  before all this.

**Comment by influenist** [comment]
Had the same. You set the material skin instead of default. Add to the addon config skin > 
material

Restart the addon and this would fix your issue.

**Comment by youseeus** [comment]
the problem here is, that the links (& images are using absolute paths (starting with a '/'). Home Assistant is serving LMS under a sub-path (like http://homeassistant.local:8123/api/hassio_ingress/jII_1UjhIHekQvX9grsOhMzMmIP11lEsUljekK1Az2g/)

So the solution for this is to store this base-path somewhere and using it to generate the links.
